templates/web/base/report/new/form_user_loggedin.html

```diff
@@ -1,20 +1,29 @@
 <div class="form-box" id="form-box--logged-in-name">
 
+[% IF js %]
+  <div style="display:none" id="js-contribute-as-wrapper">
+    [% INCLUDE form_as %]
+  </div>
+[% ELSE %]
   [% can_contribute_as_another_user = c.user.has_permission_to("contribute_as_another_user", bodies.keys.join(",")) %]
   [% can_contribute_as_body = c.user.from_body AND c.user.has_permission_to("contribute_as_body", bodies.keys.join(",")) %]
-
   [% IF can_contribute_as_another_user OR can_contribute_as_body %]
+    [% INCLUDE form_as %]
+  [% END %]
+[% END %]
+
+[% BLOCK form_as %]
```

Comment:
This is now being shown to all users when clicking the map on `/around`. It just has a single 'Yourself' option for unprivileged users. It does disappear if you refresh the `/report/new?...` URL.